tests/lean/run/reserved.lean

```diff
@@ -0,0 +1,76 @@
+def f (x : Nat) := x + 1
```

Comment:
Maybe also check what happens when the reserved names are `#check`ed or even defined before `f`.